starter-code/src/functions-and-arrays.js

```diff
@@ -11,15 +21,48 @@ var words = [
   'crackpot'
 ];
 
+function findLongestWord (words) {
+  var longest = words[0];
+  for (var i = 0; i < words.length; i++) {
+    if (words[i].length > longest.length) {
+      longest = words[i];
+    }
+  }
+  return longest;
+}
+
+
 // Calculating a Sum
 
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+function sumArray (numbers) {
+  var sum = 0;
+  for (var i = 0; i < numbers.length; i++) {
+    sum += numbers[i];
+  }
+  return sum;
+}
+
 // Calculate the Average
 
 var numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
-// Array of Strings
+function averageNumbers (numbersAvg) {
+  var total = [0];
+  for (var j = 0; j < numbersAvg.length; j++) {
+    total += numbersAvg[j];
+  }
+  if (total > 0) {
+    return total/numbersAvg.length;
+  } else if (total == 0) {
```

Comment:
You don't need this. If `total > 0` return the average. Else don't do anything. Javascript will return `undefined` as default